wiki/Performance_Points/en.md

```diff
@@ -1,156 +1,82 @@
 
-A changelog reflecting the most recent changes to the system is available [here](https://osu.ppy.sh/p/changelog?category=pp). For the obsolete ppv1 changelog, look [here](http://osu.ppy.sh/forum/t/92185).
+It aims to shift the focus of skill progression from the amount of time played to an **actual representation of the player's skill.**
 
-Location
-========
-
-The performance points ranking can be found [here](http://osu.ppy.sh/p/pp). It can also be found from the bar at the top (Rankings -&gt; Performance)
-
-To find out how the final pp is calculated, the breakdown-per-beatmap is located in "Top Ranks" tab of the profile.
+History
+=======
 
-Calculation
-===========
+Revealed to the public during April 2012 as the mysterious *'???'* testing phase, the system was eventually renamed to "pp" later on in the month. Several months after its reveal, the [(20120722-24) osu! release](http://osu.ppy.sh/forum/p/1687719) officially implemented the system to fully replace the old Ranked score system, with new scores being calculated every 30 minutes. Later on in August in the same year, the system was improved to update in real-time. 
 
-Determining the amount of points
---------------------------------
+It continued to exist in this capacity for more than a year of service, until [Tom94](http://osu.ppy.sh/u/1857058), the creator of the *osu!tp* scoring metric, joined the osu! team and implemented his design into the system. The resulting system was titled *ppv2*, and became live on the 27th of January, 2014.
 
-Performance points are heavily based on map difficulty which is determined by a unique algorithm in every gamemode. Based on this difficulty alone each of your scores is rated and assigned a pp value.
+*ppv2* is currently in active service, with live updates published to its [changelog](https://osu.ppy.sh/p/changelog?category=pp).
 
-These steps differ a lot per gamemode. Below are more detailed explanations of how each gamemodes works.
+*ppv1*, the original build of the Performance Points system also had a changelog, which you may view from its [forum topic](http://osu.ppy.sh/forum/t/92185).
 
-| ![](osu.gif "fig:osu.gif") [osu!Standard](Standard "wikilink") |-align="center" |         | (**Aim**^*X* + **Speed**^*X* + **Accuracy**^*X*)^(1/*X*) |
-|-------------------------------------------------------------------------------------------|----------------------------------------------------------|
-| ![](taiko.gif "fig:taiko.gif") [Taiko](Taiko "wikilink") |-align="center" |               | (**Strain**^*X* + **Accuracy**^*X*)^(1/*X*)              |
-| ![](ctb.gif "fig:ctb.gif") [Catch the Beat](Catch_the_Beat "wikilink") |-align="center" | | **Aim**                                                  |
-| ![](mania.gif "fig:mania.gif") [osu!mania](osu!mania "wikilink") |-align="center" |       | (**Strain**^*X* + **Accuracy**^*X*)^(1/*X*)              |
+How is pp calculated?
```

Comment:
If you rephrease this heading, you could probably remove the question mark.
`How pp is calculated`